jgrapht-core/src/main/java/org/jgrapht/alg/StrongConnectivityInspector.java

```diff
@@ -58,9 +60,10 @@
  * @author Christian Hammer
  * @since Feb 2, 2005
```

Comment:
Add a @deprecated tag to the Javadoc as well, instructing the developer to use Kosaraju instead.